helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -36,23 +39,47 @@
  * A placeholder before we have the implementation.
  * Weight-Aware Globally-Even Distribute Rebalancer.
  *
- * @see <a href="Design Document">https://github.com/apache/helix/wiki/Design-Proposal---Weight-Aware-Globally-Even-Distribute-Rebalancer</a>
+ * @see <a href="https://github.com/apache/helix/wiki/Design-Proposal---Weight-Aware-Globally-Even-Distribute-Rebalancer">
+ * Design Document
+ * </a>
  */
 public class WagedRebalancer implements GlobalRebalancer<ResourceControllerDataProvider> {
```

Comment:
Why not `GlobalRebalancer<ClusterModelDataProvider>`

Reply:
They are different. Eventually, ResourceControllerDataProvider should be ResourceDataCache or ResourceDataSnapshot.